app/core/api/views.py

```diff
@@ -156,15 +156,6 @@ class SponsorPartnerViewSet(viewsets.ModelViewSet):
     #     if is_sponsor is not None:
     #         queryset = queryset.filter(is_sponsor=is_sponsor)
     #     return queryset
-    def get_permissions(self):
-        if self.action == "create":
-            permission_classes = [
-                IsAdminUser,
-            ]
-        else:
-            permission_classes = [IsAuthenticated, IsOwnerOrReadOnly]
-        return [permission() for permission in permission_classes]
-
 
 class FaqViewSet(viewsets.ModelViewSet):
```

Comment:
This ViewSet needs API endpoint descriptions like in [this code](https://github.com/fyliu/peopledepot/blob/ee0506ddaf8ca7f09fbbadceb35d2fa361fb0a32/app/core/api/views.py#L107-L118).